compiler/luci-interpreter/include/luci_interpreter/core/Tensor.h

```diff
@@ -96,17 +95,15 @@ class Tensor
 
   const Shape &shape() const { return _shape; }
 
-  float scale() const
-  {
-    assert(_quantization.scale.size() == 1);
-    return _quantization.scale[0];
-  }
+  float scale() const { return _quantization.scale[0]; }
```

Comment:
You've removed `assert(_quantization.scale.size() == 1);`. Is this safe?

Reply:
It would be safer to leave the assert. I've updated the code. Thanks.